src/AppBundle/Entity/Post.php

```diff
@@ -95,7 +95,7 @@ class Post
      *      mappedBy="post",
      *      orphanRemoval=true
      * )
-     * @ORM\OrderBy({"publishedAt" = "DESC"})
+     * @ORM\OrderBy({"publishedAt": "DESC"})
```

Comment:
@frastel FrameworkBundle does not have annotations. It uses doctrine/annotations.

The difference you see corresponds to 2 different features of the annotation syntax.
Putting a value in a property of the annotation and writing an associative array in a value are already 2 different things.
what could confuse you is that the property name is omitted here (thanks to a feature allowing to do it in some cases). The complete syntax is ``@ORM\OrderBy(value={"publishedAt": "DESC"})``